Azure/sonic-buildimage
# Enable SAI_SWITCH_ATTR_UNINIT_DATA_PLANE_ON_REMOVAL attribute

Pass fast reboot option to syncd so that SAI_SWITCH_ATTR_UNINIT_DATA_PLANE_ON_REMOVAL gets triggered

Signed-off-by: Thushar Gowda <24815472+tbgowda@users.noreply.github.com>


#### Why I did it

Fixes https://github.com/Azure/sonic-buildimage/issues/8980 partly.

The corresponding changes in sonic-sairedis is here : 
https://github.com/Azure/sonic-sairedis/pull/975

#### How I did it
Include changes from both repos and build an image for verification.

#### How to verify it
Trigger fast-reboot with the changes, see the attribute SAI_SWITCH_ATTR_UNINIT_DATA_PLANE_ON_REMOVAL being set at the SAI level.

#### Which release branch to backport (provide reason below if selected)
Ideally it is preferred we back port to all branches where fast-reboot is supported. However, 202012 seems like a critical release train.


#### Description for the changelog
Enable SAI_SWITCH_ATTR_UNINIT_DATA_PLANE_ON_REMOVAL in case of fast-reboot
